symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkRetryConfig.java

```diff
@@ -19,6 +19,31 @@
     private double multiplier = DEFAULT_MULTIPLIER;
     private long maxIntervalMillis = DEFAULT_MAX_INTERVAL_MILLIS;
 
+  /**
+   * For testing purposes only.
```

Comment:
If for testing only, could we instead provide a utility class available from test sources only? Something like `BdkRetryConfigTestHelper#ofMinimalInterval`